A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A320_Neo_MFD.js

```diff
@@ -853,16 +853,23 @@ class A320_Neo_MFD_NDInfo extends NavSystemElement {
         }
         const ADIRSState = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum");
         const groundSpeed = Math.round(Simplane.getGroundSpeed());
+        const trueAirSpeed = SimVar.GetSimVarValue("AIRSPEED TRUE", "knots");
+        const windStrength = SimVar.GetSimVarValue("AMBIENT WIND VELOCITY", "knots");
         const gs = this.ndInfo.querySelector("#GS_Value");
         const tas = this.ndInfo.querySelector("#TAS_Value");
         const wd = this.ndInfo.querySelector("#Wind_Direction");
         const ws = this.ndInfo.querySelector("#Wind_Strength");
         const wa = this.ndInfo.querySelector("#Wind_Arrow");
         const wptg = this.ndInfo.querySelector("#Waypoint_Group");
         const wptPPOS = this.ndInfo.querySelector("#Waypoint_PPOS");
-        if (ADIRSState != 2 || groundSpeed <= 100) {
-            //Hide TAS, and wind info
+        if (ADIRSState != 2 || trueAirSpeed < 60) {
+            //TAS info Conditions
             tas.textContent = "---";
+        } else {
+
+        }
+        if (ADIRSState != 2 || trueAirSpeed < 100 || windStrength <= 2) {
```

Comment:
**suggestion**: rename `windStrength` to `windVelocity`.

Although _wind force_ exists (Beaufort scale), _wind strength_  doesn't as far as I know. As the unit is knots using `windVelocity` or `windVelocityInKnots` makes more sense.

Reply:
I decided to take WindStrength as the naming, since the other NDInfos for the wind are labeled as wind Strength aswell so i thought to keep it uniformly i name it that aswell. Let me know what you think and thanks for the review :)